packages/react-native/React/Views/ScrollView/RCTScrollView.m

```diff
@@ -338,6 +338,12 @@ - (void)_keyboardWillChangeFrame:(NSNotification *)notification
     if (!didFocusExternalTextField && focusEnd > endFrame.origin.y) {
       // Text field active region is below visible area with keyboard - update diff to bring into view
       contentDiff = endFrame.origin.y - focusEnd;
+    } else {
+      UIView *inputAccessoryView = _firstResponderViewOffScrollView.inputAccessoryView;
+      if (inputAccessoryView) {
+        // Text input view is within the inputAccessoryView.
+        contentDiff = endFrame.origin.y - beginFrame.origin.y;
+      }
```

Comment:
I'm surprised the logic for Fabric and Paper has diverged here quite a bit - should this be unified?